warehouse/seeds/ntd_agency_to_organization.csv

```diff
@@ -229,3 +229,25 @@ A0013,,,
 9R02-91032,9R02-143,recbW86Xrtuw8PhiU,City of Auburn
 90026,9026,recZALk4vysuoTVjF,San Diego Metropolitan Transit System
 90278,,,
+90270,,rec860cgWfE3MaiHw,Los Angeles County
+90271,,rec860cgWfE3MaiHw,Los Angeles County
+90272,,rec860cgWfE3MaiHw,Los Angeles County
+90276,,rec860cgWfE3MaiHw,Los Angeles County
+90277,,rec860cgWfE3MaiHw,Los Angeles County
+90269,,rec860cgWfE3MaiHw,Los Angeles County
+90273,,rec860cgWfE3MaiHw,Los Angeles County
+90274,,rec860cgWfE3MaiHw,Los Angeles County
+90275,,rec860cgWfE3MaiHw,Los Angeles County
+90278,,rec860cgWfE3MaiHw,Los Angeles County
+90279,,rec860cgWfE3MaiHw,Los Angeles County
+9R02-99442,,recFqVabH8109u70q,Calaveras Transit Agency
```

Comment:
If more mappings have been added manually, we may need to update here: https://github.com/cal-itp/data-infra/blob/main/warehouse/seeds/_seeds.yml#L86 to describe what versions of NTD data are being used to make these mappings.

Reply:
I don't know how the initial seeds were created, so I don't know what to add here.

Reply:
I added a note in [e52aa76](https://github.com/cal-itp/data-infra/pull/2404/commits/e52aa76163db893d4d374fdced1715beafc45ec5)